### ISSUE TITLE
BLR educational chanel on youtube of vimeo?

### ISSUE THREAD

**Issue by myrmoteras** [plazi/arcadia-project#131]
we want to build an educational channel for BLR on either vimeo or youtube to teach how to use Plazi software, but probably more general about the liberation activities.

The question is, whether this should be run on youtube or vimeo. What are the pros and cons? 

We also need to consider to integrate this into the new website.

here two example of movies:
1. plazi liberation promotion https://vimeo.com/358815174
2. possible video on mark up (this will be an entire series documenting the entire markup process step by step https://youtu.be/CM4YeL5y7A8

**Comment by myrmoteras** [comment]
via @punkish 
Vimeo is more professional, supports only better and higher quality, has no ads, but has a smaller audience. YT is more popular, supports low quality as well, is full of ads and Google's suggestion algorithm, and has a gigantic audience

Vimeo is where you find more artsy and professional stuff. YT is where you find a lot of good stuff but also a lot of crap

We have a small audience and they are all targeted, so I would prefer Vimeo. On YT we are likely to get lost (small fish in a very, very big ocean)

**Comment by gsautter** [comment]
What, exactly, keeps us from uploading whatever video we make to _both_ platforms? I don't really set the "either-or" here, to be honest ...

**Comment by myrmoteras** [comment]
![image](https://user-images.githubusercontent.com/4609956/70221590-f0027b00-1748-11ea-8530-2c7a530cf5fe.png)

![image](https://user-images.githubusercontent.com/4609956/70221604-f5f85c00-1748-11ea-87aa-2a4d55094168.png)

![image](https://user-images.githubusercontent.com/4609956/70221631-03ade180-1749-11ea-821f-212318b1dfad.png)

![image](https://user-images.githubusercontent.com/4609956/70221659-0c9eb300-1749-11ea-8f17-007acf3f415f.png)
![image](https://user-images.githubusercontent.com/4609956/70221735-2d670880-1749-11ea-8610-706a594fb8da.png)

default YT view of a video (why is Warren Buffet and "Life at Google" being recommended to me? Why is *anything* being recommended to me?)

., [05.12.19 10:11]
one is sophisticated, the other is crowded and noisy

**Comment by punkish** [comment]
> What, exactly, keeps us from uploading whatever videos we make to _both_ platforms? I don't really see the "either-or" here, to be honest ...

That is a very good point, and yes, we could simply maintain two parallel channels. 

That said, there might be both direct costs (Vimeo subscription; on YT I don't know how channels work when they don't have any ads allowed on them, and I also don't know whether YT allows high def videos without charging anything for that privilege) and indirect costs (simply the internal cost of uploading to two different platforms, keeping them both in sync, etc.)

**Comment by gsautter** [comment]
Well, we might use Vimeo as the primary channel (and link to those videos whenever linking videos), and run one on YT on the side, maybe even allowing ads ... would be funny to see what kinds of ads actually come up for our kind of content.
Regarding keeping things in sync, I don't really see all too much effort there, as I wouldn't tend to think that we'll have to change all too much in a video once it's uploaded ... or do we?

**Comment by mguidoti** [comment]
I didn't explore much but I also haven't seen any restrictions on video quality or to keep ads out of our vids on YT. I don't know how the analytics of Vimeo works, what kind of stats they make available to us, but I wouldn't be surprise that is something very similar as well. Honestly, I think for our use both platforms will be very similar and it'll be a matter of personal preference, I suppose. 

Whatever you guys decide will be fine for us. We'll keep posting these drafts privately on YT, as they're not the final versions and they're intended to collect feedback from you guys only. If even these drafts must be moved to somewhere else, just let us know.

Basically, whenever your final decision is made, we'll adapt and move forward accordingly.

**Comment by punkish** [comment]
I lean toward Vimeo. That said, I think @gsautter has a good idea. Let's do both Vimeo and YT. If it really *only* involves uploading stuff, perhaps it is not that great a burden. Personally, I don't give a rat's ass about viewer stats, but that is just me. There is one impt thing though – as far as I know, YT doesn't allow downloading videos for offline viewing (it is a part of their TOS). Vimeo, on the other hand, does allow downloading videos if the video owner so desires. That, to me is a very impt difference and a plus point for Vimeo.
